### ISSUE TITLE
is there any plan to provide good documentation for the library that covers most aspects ?

### ISSUE THREAD

**Issue by Aneeqasif** [marcoheisig/Petalisp#23]

**Comment by marcoheisig** [comment]
Yes, good documentation is already underway.  I have a draft of a user manual with about 80 pages that I'll upload soon.  The reason why that manual hasn't been published so far is that it is part of my PhD thesis.  Give me a few more weeks to wrap that up.

**Comment by Aneeqasif** [comment]
> Yes, good documentation is already underway. I have a draft of a user manual with about 80 pages that I'll upload soon. The reason why that manual hasn't been published so far is that it is part of my PhD thesis. Give me a few more weeks to wrap that up.

thats great i am excited to hear that. looking forward

**Comment by marcoheisig** [comment]
Status update: I made several major improvements to Petalisp, especially in terms of broadcasting semantics and scheduling, which delayed the whole endeavor by a few months.  At the same time, the manual has grown to almost 200 pages.  I am now in the final sweep of proofreading, and I'll publish the whole thing in October.

Thanks everyone for your patience!
